19.0/Dockerfile

```diff
@@ -45,17 +47,22 @@ ENV BUILD_PACKAGE="\
 # Default SHELL is ["/bin/sh", "-c"]
 SHELL ["/bin/sh", "-e", "-x", "-c"]
 
+
 # Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
 RUN /install/package_odoo.sh \
```

Comment:
We can save a few (although very small) layers with:

```dockerfile
RUN --mount=type=bind,src=./base_requirements.txt,dst=/odoo/base-requirements.txt \
    --mount=type=bind,src=./extra-requirements.txt,dst=/odoo/extra-requirements.txt \
```

and so we don't need to COPY this files into the image, cause they're only used at build time